training_toolbox/lpr/README.md

```diff
@@ -0,0 +1,77 @@
+# LPRNet: License Plate Recognition 
+
+
+## Train LPRNet model
+
+To train a [LPRNet](https://arxiv.org/abs/1806.10447), jump to
+training_toolbox/lpr directory. You'll see the folder with sample code
+demonstrating how to train a LPRNet model.
+
+We provide predefined configuration for: 
+* Chinese license plates recognition.
+  - Configuration file: [training_toolbox/lpr/chinese_lp/config.py](chinese_lp/config.py).
+  - Trained model: [LPRNet 94x24](https://download.01.org/openvinotoolkit/training_toolbox_tensorflow/models/lpr/chinese_lp/license-plate-recognition-barrier-0007.zip).
+
+As training dataset for this model [Synthetic Chinese License Plates](https://download.01.org/openvinotoolkit/training_toolbox_tensorflow/datasets/Synthetic_Chinese_License_Plates.tar.gz) dataset was used.
+  
+To train a model, go through the following steps:
+
+1. Download training data and extract it in some `<dataset_folder>`. It will consist from
+    folder with training images named `crops` and text file with annotations named 
+    `annotation`. 
+    
+    After extracting training data archive run python script from 
```

Comment:
We have a directory with datasets: `data`. I think it is a good practice to store all datasets and processing scripts in one place.